electron/src/preload.ts

```diff
@@ -1,14 +1,15 @@
+// tslint:disable-next-line: no-var-requires
 const { ipcRenderer } = require("electron")
 const electronProcess = process
 
 let nextCommandID = 1
 
-function createCommand(commandType, eventType) {
-  return (...args) => {
+function createCommand<T>(commandType: string, eventType: string): () => Promise<T> {
```

Comment:
We could also type the command function arguments here (as another generic type param). But since we are gonna get rid of this function in the course of #796, anyway, this is totally fine with me as is.